src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -309,7 +309,7 @@ public async Task ErrorListInfoBarController_Refresh_CriticalExceptionIsNotSuppr
             var testSubject = CreateTestSubject(host);
 
             // Act
-            var act = async () => await testSubject.ProcessSolutionBindingAsync();
+            Func<Task> act = async () => await testSubject.ProcessSolutionBindingAsync();
```

Comment:
Fixing a test compilation error from a previous commit.

The test projects are set to use the `latest` C# language version. Building with VS2022/MSBuild v17.0 means we can use `var` here i.e. on our dev machines.

However, the CI machine is still using MSBuild v16.0, so we can't use `var` here.

Reply:
Ah, so that's why it failed. Good to know.